.github/workflows/go.yml

```diff
@@ -0,0 +1,29 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+    - name: Test
+      run: go test ./... -short
```

Comment:
It would be great to use build tags and split integration and unit tests.
`Short` basically used to turn off long running tests

Reply:
You are right. I known it, but for it integration and unit tests must be splitted into other files. So I decided to organize it with usage _Short_. It is convenient when all tests are located in the same file.